cmd/init.js

```diff
@@ -39,12 +39,22 @@ module.exports = (ipc) => async function init (cmd) {
   try {
     await output(false, await require('../init')(link, dir, { ipc, autosubmit: yes, force, defaults, header }))
   } catch (err) {
-    if (err.code !== 'ERR_PERMISSION_REQUIRED') throw err
-    const explain = 'Be sure that software is trusted before using it\n' +
-    '\nType "TRUST" to allow template initialization or anything else to exit\n\n'
-    const ask = 'Trust template'
-    const act = 'Use pear init again to initalize from trusted template'
-    await trust({ ipc, key: err.info.key, message: err.message, explain, ask, act })
+    if (err.code !== 'ERR_PERMISSION_REQUIRED') {
+      throw err
+    } else {
```

Comment:
if you throw you exit the function - no need for the else branch - also means you can de-nest 
```
if (err.code !== 'ERR_PERMISSION_REQUIRED') throw err
//rest of code at this indent
```

Reply:
updated